### ISSUE TITLE
Aggregate Menu

### ISSUE THREAD

**Issue by palexdev** [home-sweet-gnome/dash-to-panel#1624]
I recently updated to Gnome 42. I was previously using Dash to Dock but since it's not been updated yet and updates were starting to accumulate (270+ packages according to pamac lol) I decided to use Dash to Panel. Of course I also had to fix other extensions and also switch to a theme which supports Gnome 42.
So this is my setup now: [Imgur](https://imgur.com/a/3Qj25qY)

There's an issue though. I can't see the volume percentage anywhere. I could use the Sound Percentage extension but the aggregate menu changes only on the top bar while the one on Dash to Panel is always the default one. I was also using some extensions to change the source and change the volume per app but they don't show on Dash to Panel as well.
(Also note that the I hidden the aggregate menu on the top menu since I'd like to have it only on Dash to Panel, similar to a Windows setup)

Can something be done?

**Comment by benintech** [comment]
+1 here
I am using the same kind of setup as you : leave the original panel unchanged, and a dash-to-panel at the bottom.

I think the issue can be broadened : if you choose to display in the dash panel items from the top panel, like the aggregate menu or the date menu, other extensions that alter those items are not applied to the dash panel.

I moved the aggregate menu (like you) but also the date menu from the top panel to the dash panel. Extensions like « [Hibernate Status Button](https://extensions.gnome.org/extension/755/hibernate-status-button/) » (which adds an « Hibernate » option to the aggregate menu) and « [Panel Date Format](https://extensions.gnome.org/extension/3465/panel-date-format/) » (which modifies the format ot the clock) are indeed applied to the top panel, but not the the dash panel.

Basically, it seems that if you choose to move those items to the dash panel, you are stuck with their original form, other extensions will not be applied to those moved items.

Anyone, any chance to change this ?

**Comment by benintech** [comment]
Stale bot : should not be automatically closed without some input of the devs.
Devs should say if this limitation has a chance to be fixed in the future, or should close the issue manually if nothing will be done about it.
